screens/SignUp.js

```diff
@@ -50,6 +53,26 @@ function SignIn({ navigation }) {
       setErrorMessage("Passwords do not match - please re-enter");
       return;
     }
+    try {
+      setLoading(true);
+      const player = await DataStore.save(
+        new Player({
+          name: name,
+          skill_level: 5,
```

Comment:
Should skill_level still be a number? or a string - maybe "Beginner" and we will eventually add a way to change it? Or maybe "Any"?